index.bs

```diff
@@ -206,268 +177,455 @@ interface ImageCapture {
+  <dd>Whether <a>brightness</a> constraining is recognized.</dd>
 
-  <dt><dfn dict-member for="PhotoSettings"><code>brightness</code></dfn></dt>
-  <dd>This reflects the desired <a>brightness</a> setting of the camera.</dd>
+  <dt><dfn dict-member for="MediaTrackSupportedConstraints"><code>contrast</code></dfn></dt>
+  <dd>Whether <a>contrast</a> constraining is recognized.</dd>
 
-  <dt><dfn dict-member for="PhotoSettings"><code>contrast</code></dfn></dt>
-  <dd>This reflects the desired <a>contrast</a> setting of the camera.</dd>
+  <dt><dfn dict-member for="MediaTrackSupportedConstraints"><code>saturation</code></dfn></dt>
+  <dd>Whether <a>saturation</a> constraining is recognized.</dd>
 
-  <dt><dfn dict-member for="PhotoSettings"><code>saturation</code></dfn></dt>
-  <dd>This reflects the desired <a>saturation</a> setting of the camera.</dd>
+  <dt><dfn dict-member for="MediaTrackSupportedConstraints"><code>sharpness</code></dfn></dt>
+  <dd>Whether <a>sharpness</a> constraining is recognized.</dd>
 
-  <dt><dfn dict-member for="PhotoSettings"><code>sharpness</code></dfn></dt>
-  <dd>This reflects the desired <a>sharpness</a> setting of the camera.</dd>
+  <dt><dfn dict-member for="MediaTrackSupportedConstraints"><code>zoom</code></dfn></dt>
+  <dd>Whether configuration of the <a>zoom</a> level is recognized.</dd>
 
-  <dt><dfn dict-member for="PhotoSettings"><code>zoom</code></dfn></dt>
-  <dd>This reflects the desired zoom setting of the camera.</dd>
+  <dt><dfn dict-member for="MediaTrackSupportedConstraints"><code>torch</code></dfn></dt>
+  <dd>Whether configuration of <a>torch</a> is recognized.</dd>
+</dl>
 
-  <dt><dfn dict-member for="PhotoSettings"><code>imageWidth</code></dfn></dt>
-  <dd>This reflects the desired image height.  The UA MUST select the closest height value this setting if it supports a discrete set of height options.</dd>
+## {{MediaTrackCapabilities}} dictionary ## {#mediatrackcapabilities-section}
 
-  <dt><dfn dict-member for="PhotoSettings"><code>imageHeight</code></dfn></dt>
-  <dd>This reflects the desired image width. The UA MUST select the closest width value this setting if it supports a discrete set of width options.</dd>
+{{MediaTrackCapabilities}} is extended here with the capabilities specific to image capture. This dictionary is produced by the UA via {{getCapabilities()}} and represents the supported ranges and enumerations of the supported constraints.
 
-  <dt><dfn dict-member for="PhotoSettings"><code>fillLightMode</code></dfn></dt>
-  <dd>This reflects the desired fill light (flash) mode setting.  Acceptable values are of type {{FillLightMode}}.</dd>
+<pre class="idl">
+  partial dictionary MediaTrackCapabilities {
+    sequence&lt;MeteringMode>  whiteBalanceMode;
+    sequence&lt;MeteringMode>  exposureMode;
+    sequence&lt;MeteringMode>  focusMode;
+
+    MediaSettingsRange      exposureCompensation;
```

Comment:
Yeah I lobbied for `ideal` to mean default value in capabilities long ago in mediacapture, but there was no interest at the time. Shouldn't disrupt anything IMHO.